apps/rush-lib/src/api/LastInstallFlag.ts

```diff
@@ -1,8 +1,12 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
+// eslint-disable-next-line @typescript-eslint/typedef
+const importLazy = require('import-lazy')(require);
+
 import * as path from 'path';
-import * as _ from 'lodash';
+// eslint-disable-next-line @typescript-eslint/typedef
```

Comment:
@sachinjoseph Looking at this more closely, I realize that everywhere we call `importLazy()` we lose all type checking for those APIs throughout the whole file. That is a concern. I'm going to investigate this today and see if there is some way to fix this.

@iclanton